db/database.js

```diff
@@ -0,0 +1,37 @@
+const mongoose = require('mongoose');
+
+mongoose.connect('mongodb://localhost/fetcher');
```

Comment:
i was just messing around with this url, doesn't seem like the fetcher changed anything to be honest. may not be a good idea to use fetcher here as well since this is setting up a mock environment and don't want the chances to somehow mess with the real DB